src/drivers/St7789.cpp

```diff
@@ -179,14 +178,15 @@ void St7789::DrawBuffer(uint16_t x, uint16_t y, uint16_t width, uint16_t height,
 }
 
 void St7789::HardwareReset() {
-  nrf_gpio_pin_clear(26);
+  nrf_gpio_pin_clear(pinReset);
   nrf_delay_ms(10);
-  nrf_gpio_pin_set(26);
+  nrf_gpio_pin_set(pinReset);
 }
```

Comment:
Can we configure the pin only at the beginning of this function, and reset back to default before returning?

Reply:
The pins needs to be kept at a high level to ensure the display controller won't reset unexpectedly, so I guess we can't configure the pins to default (input, no pull up/down) here.

Reply:
If that's true, wouldn't that mean we shouldn't default configure the pin in the sleep function either or anywhere else?

Reply:
Yes, you are right, I fixed this in 3c668ca01bb95e02dadb7a9bb1e7f43ddf9071e5.

Reply:
No, there's still nrf_gpio_cfg_default in the St7789::Sleep() function.

Reply:
Yes, right, I guess we shouldn't default the reset pin!